.github/workflows/linux-checks.yaml

```diff
@@ -30,10 +29,17 @@ jobs:
         uses: arduino/setup-protoc@master
       - name: Add bin to path
         run: echo "::add-path::${{env.GOPATH}}/bin"
+      - name: Go mod cache setup
+        uses: actions/cache@v2
+        with:
+          path: ${{env.GOPATH}}/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
       - name: Fetch vendor and check succesful build
         run: |
-          go mod download
-          make vendor install-build-tools pre-build proto build
+          make vendor
+          make install-build-tools pre-build proto build
```

Comment:
Why move vendor target to a separate line?

Reply:
fixed